src/main/java/seedu/address/model/person/lead/Lead.java

```diff
@@ -36,6 +36,27 @@ public static boolean isValidLead(String lead) {
                 || lead.equalsIgnoreCase("COLD");
     }
 
+    /**
+     * Returns true if the lead is hot.
+     */
+    public boolean isHot() {
+        return leadType.equals(LeadType.HOT);
+    }
+
+    /**
+     * Returns true if the lead is warm.
+     */
+    public boolean isWarm() {
+        return leadType.equals(LeadType.WARM);
+    }
+
+    /**
+     * Returns true if the lead is cold.
+     */
+    public boolean isCold() {
+        return leadType.equals(LeadType.COLD);
+    }
+
```

Comment:
I wonder if there's a better way to do this so it's more flexible since this logic is repeated three times already.